### PR TITLE
Write to db

### PR DESCRIPTION
**Pull request by braaannigan**

### CODE REVIEW

**Comment by braaannigan** [comment]
@c-peters I'm not sure how to write the code snippet to fix this with the new format - could you have a look please?

File: docs/src/python/user-guide/io/database.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"""
+f"""
```
Comment (c-peters) [inline docs/src/python/user-guide/io/database.py:1]: This should be a regular multi line string, not an f-string

File: user_guide/src/SUMMARY.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Summary
```
Comment (c-peters) [inline user_guide/src/SUMMARY.md:1]: This file is not in use anymore

File: user_guide/src/howcani/io/read_db.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Read from a database
```
Comment (c-peters) [inline user_guide/src/howcani/io/read_db.md:1]: This file is not in use anymore

File: user_guide/src/SUMMARY.md (hunk 2 above)
Comment (braaannigan) [reply user_guide/src/SUMMARY.md:1]: Cheers, these files were just left over on my branch as they were open in VScode, I'll rm them

File: docs/src/python/user-guide/io/database.py (hunk 1 above)
Comment (braaannigan) [reply docs/src/python/user-guide/io/database.py:1]: Change made

**Comment by braaannigan** [comment]
Thanks @c-peters!

**Pull request merged**